app/src/main/java/com/android/sample/ui/testScreens/RecipeScreen.kt

```diff
@@ -1,15 +1,239 @@
+                    text = "${Random().nextFloat() * 3 + 2}",
+                    style = MaterialTheme.typography.bodyMedium)
+              }
+              Row(verticalAlignment = Alignment.CenterVertically) {
+                // recipe time
+                Icon(
+                    imageVector = Icons.Filled.AccessTime,
+                    contentDescription = "rating",
+                    modifier = Modifier.size(16.dp).testTag("recipeTimeIcon${recipe.idMeal}"),
+                    tint = Color.Gray)
+                Text(
+                    modifier = Modifier.padding(4.dp).testTag("recipeTime${recipe.idMeal}"),
+                    text =
+                        "${arrayOf(
+                              10,
+                              15,
+                              20,
+                              30,
+                              45,
+                              50
+                          ).random()} min",
+                    style = MaterialTheme.typography.bodyMedium)
+
+                Spacer(modifier = Modifier.width(8.dp))
+
+                // recipe cost
+                Price(cost = Random().nextInt(3), recipe = recipe)
+              }
+            }
+      }
+}
+
+/**
+ * Recipe list composable that displays a list of recipes.
+ *
+ * @param userViewModel view model for user data
+ * @param navigationActions navigation actions
+ */
+@Composable
+fun RecipeList(userViewModel: UserViewModel, navigationActions: NavigationActions) {
+  var searchText by remember { mutableStateOf("") }
+  val recipesList: List<Recipe> = userViewModel.likedRecipes.collectAsState().value
   Scaffold(
-      modifier = Modifier.testTag("recipeScreen"),
-      content = { Text(text = "recipeScreenText", modifier = Modifier.padding(it)) })
+      modifier = Modifier.testTag("recipeList"),
+      containerColor = Color(0xFFFFFFFF),
+      topBar = {
+        Column(
+            verticalArrangement = Arrangement.spacedBy(0.dp, Alignment.Top),
+            horizontalAlignment = Alignment.Start) {
+              Button(
+                  onClick = {},
+                  colors =
+                      ButtonDefaults.buttonColors(containerColor = Color.White), // Button color
+                  shape = RoundedCornerShape(50), // Circular edges for the button
+                  border = BorderStroke(1.dp, Color.LightGray),
+                  modifier =
+                      Modifier.padding(start = 16.dp, top = 8.dp, bottom = 16.dp)
+                          .height(64.dp)
+                          .width(329.dp)
+                          .testTag("SearchButton")) {
+                    Row(modifier = Modifier.fillMaxWidth()) {
+                      Icon(
+                          modifier = Modifier.testTag("searchButtonIcon"),
+                          imageVector = Icons.Filled.Search,
+                          contentDescription = "Search",
+                          tint = Color.Gray)
```

Comment:
For colors, like the tint here or the color of the background, please use the ones given in the theme.

Reply:
I agree with that. I'll make sure to use the colors provided in the theme for things like tints and background colors.